Content.Client/OmniPainter/UI/OmniPainterBoundUserInterface.cs

```diff
@@ -0,0 +1,70 @@
+using Content.Shared.OmniPainter;
+using Robust.Client.GameObjects;
+using Robust.Client.UserInterface.Controls;
+
+namespace Content.Client.OmniPainter.UI
```

Comment:
since rename affected most of the lines why not also replace scoped namespaces with file namespaces